admin/tools/docker/latest/Dockerfile

```diff
@@ -47,22 +26,26 @@ ARG EUPS_TAG
 # Install development and debugging tools
 #
 RUN if [ "$EUPS_TAG" = "qserv-dev" ] ; then \
-    apt-get -y install byobu dnsutils \
-        gdb graphviz lsof \
-        net-tools vim ; \
+        yum install -y \
```

Comment:
Ideally, it would be also nice to include sysstat (for performance monitoring) to have tools like "iostat" inside the container. Some host deployments may not have those tools by default. Though, I don't think it's terribly critical here.

Reply:
Systat (and vim) are already in the base image here, so we’re covered!